--- a/setup.py
+++ b/setup.py
@@ -982,7 +982,8 @@ def detect_simple_extensions(self):
 
         # Lance Ellinghaus's syslog module
         # syslog daemon interface
-        self.addext(Extension('syslog', ['syslogmodule.c']))
+        # Termux: Add 'log' android library since we use android logging
+        self.addext(Extension('syslog', ['syslogmodule.c'], libraries=['log']))
 
         # Python interface to subinterpreter C-API.
         self.addext(Extension('_xxsubinterpreters', ['_xxsubinterpretersmodule.c']))
